### PR TITLE
FPLA-3208: Trigger Welsh translations for C110A

### PR DESCRIPTION
**Pull request by samuelzerozero**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-3208
### Change description ###
FPLA-3208: Trigger Welsh translations for C110A
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by TSeelig — APPROVED** [review]

File: ccd-definition/AuthorisationCaseField/CareSupervision/system-update.json [hunk 1]
```diff
@@ -272,6 +293,34 @@
...
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "submittedFormNeedTranslation",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRU"
+  },
```
[the remark this answers is not in the document]
Comment (samuelzerozero) [reply ccd-definition/AuthorisationCaseField/CareSupervision/system-update.json:323]: systemupdate is the system user (doesn't have ui, and the label is not stored in caseData)

File: ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json [hunk 2]
```diff
@@ -951,6 +951,27 @@
...
+    "UserRole": "caseworker-publiclaw-magistrate",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "translatedSubmittedForm",
+    "UserRole": "caseworker-publiclaw-magistrate",
+    "CRUD": "R"
+  },
```
[the remark this answers is not in the document]
Comment (samuelzerozero) [reply ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json:974]: good spot, adding it now

File: ccd-definition/CaseEvent/CareSupervision/MultiState.json [hunk 3]
```diff
@@ -402,6 +402,7 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/language-selection/about-to-submit",
```
[the remark this answers is not in the document]
Comment (samuelzerozero) [reply ccd-definition/CaseEvent/CareSupervision/MultiState.json:405]: we don't know with which code is going to fail :(

**Review by Rob-Doherty — APPROVED** [review]
looks good to me

**Pull request merged**